perf/benchmark/configs/run_perf_test.conf

```diff
@@ -1,6 +1,9 @@
 none=true
 none_tcp=true
 plaintext=false
+security_authz=true
```

Comment:
This PR is only adding peer authn and request authn right? If so remove the `security_authz=true` line (should be added in the other PR for authz)